### PR TITLE
 expvar flag compatibilti fix

### PR DESCRIPTION
**Pull request by trueival**
hot fix for compatibility with old yandex-tank versions

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -90,10 +90,16 @@ func Run() {
 	}
 	var (
 		example bool
+		expvar  bool
 	)
 	flag.BoolVar(&example, "example", false, "print example config to STDOUT and exit")
+	flag.BoolVar(&expvar, "expvar", false, "stab for compatibility")
```
Comment (direvius) [inline cli/cli.go]: "enable expvar service (DEPRECATED, use monitoring config section instead)"

File: cli/cli.go [hunk 2]
```diff
@@ -90,10 +90,16 @@ func Run() {
...
 	flag.Parse()
 
+	if expvar {
+		fmt.Fprintf(os.Stderr, "-expvar flag is a stab. Use monitoring section in config to turn it on\n")
```
Comment (direvius) [inline cli/cli.go]: "is DEPRECATED"

Comment (trueival) [reply cli/cli.go]: fixed

File: cli/cli.go (hunk 1 above)
Comment (trueival) [reply cli/cli.go]: fixed

**Pull request merged**